include/mbedtls/ssl.h

```diff
@@ -107,6 +107,7 @@
 #define MBEDTLS_ERR_SSL_TIMEOUT                           -0x6800  /**< The operation timed out. */
 #define MBEDTLS_ERR_SSL_CLIENT_RECONNECT                  -0x6780  /**< The client initiated a reconnect from the same port. */
 #define MBEDTLS_ERR_SSL_UNEXPECTED_RECORD                 -0x6700  /**< Record header looks valid but is not expected. */
+#define MBEDTLS_ERR_SSL_IGNORE_NON_FATAL                  -0x6680  /**< The message is a non-fatal error. */
```

Comment:
The context here is the TLS standard and it specifies which alert messages are fatal and which are not. It cannot be summarised in one line and I don't want to reference the standard here specifically, because then we should reference it at almost every line if we want to stay consistent.